src/Microsoft.DotNet.ProjectModel/Project.cs

```diff
@@ -84,6 +84,8 @@ public string ProjectDirectory
 
         public bool IsTestProject => !string.IsNullOrEmpty(TestRunner);
         
+        public bool OverrideIsRunnable { get; set; }
```

Comment:
Because that option doesn't go into project.json anymore.  It goes into .csproj <OutputType>Exe</>

---
In reply to: [71797075](https://github.com/dotnet/cli/pull/3911#discussion_r71797075) [](ancestors = 71797075)